### PR TITLE
Update text

### PR DESCRIPTION
**Pull request by jinnyu**

### CODE REVIEW

**Comment by spencerjunraid** [comment]
Thank you for the suggestions. I will need to find someone to review before merging.

**Comment by spencerjunraid** [comment]
Still reviewing...

File: translations.txt [hunk 1]
```diff
@@ -178,7 +178,7 @@ NA=NA
 Name=名称
 Network=网络
 Never=永不
-New device, in standby mode spun-down=新设备，处于待机模式（旋转速度下降）
+New device, in standby mode spun-down=新设备，处于待机模式
```
Comment (spencerjunraid) [inline translations.txt:181]: Please update to 新设备，处于待机模式（已降速）per @fss003124

File: tools.txt [hunk 2]
```diff
@@ -31,7 +31,7 @@ Display Settings=显示设置
 End-User License Agreement=最终用户许可协议
 Enter URL of language pack XML file=输入语言包 XML 文件的 URL 地址
 Error accessing your physical USB Flash boot device=访问物理 USB 闪存引导设备时出错
-EULA=EULA
+EULA=终端用户许可协议
```
Comment (spencerjunraid) [inline tools.txt:34]: Please change to 最终用户许可协议 per @fss003124's review.

File: settings.txt [hunk 3]
```diff
@@ -180,7 +180,7 @@ exceeds Disk Shutdown=超过磁盘关闭次数
 Excluded disks=排除的磁盘
 Execution error=执行错误
 Failed to stop=停止失败
-Failure to do so may result in your docker image suffering corruption at a later time=否则可能导致 Docker 映像日后损坏
+Failure to do so may result in your docker image suffering corruption at a later time=无法进行操作, 稍后使用 Docker 镜像时可能出错.
```
Comment (spencerjunraid) [inline settings.txt:183]: Please change to 如未能进行此操作可能会导致 Docker 映像日后遭受损坏

File: settings.txt [hunk 4]
```diff
@@ -87,17 +87,17 @@ Custom banner=自定义横幅
 Custom UPS cable=自定义 UPS 电缆
 Custom=自定义
 custom=自定义
-D comma G dot=D 逗点：G 点
-D comma G none=D 逗点：G 无
-D comma G space=D 逗点：G 空格
+D comma G dot=D 逗号：G 点
```
Comment (spencerjunraid) [inline settings.txt:90]: good

File: settings.txt [hunk 5]
```diff
@@ -87,17 +87,17 @@ Custom banner=自定义横幅
...
+D comma G none=D 逗号：G 无
```
Comment (spencerjunraid) [inline settings.txt:91]: good

File: settings.txt [hunk 6]
```diff
@@ -87,17 +87,17 @@ Custom banner=自定义横幅
...
+D comma G space=D 逗号：G 空格
```
Comment (spencerjunraid) [inline settings.txt:92]: good

File: settings.txt [hunk 7]
```diff
@@ -108,7 +108,7 @@ Day, YYYY-MM-DD=日期，年-月-日
 DD-MM-YYYY=日-月-年
 Default appdata storage location=默认应用数据存储位置
 Default critical disk temperature threshold=默认危险磁盘温度阈值
-Default critical disk utilization threshold=默认危险磁盘利用率阈值
+Default critical disk utilization threshold=默认危险磁盘容量阈值
```
Comment (spencerjunraid) [inline settings.txt:111]: Keep original translation please.

File: settings.txt [hunk 8]
```diff
@@ -117,13 +117,13 @@ Default SMART attribute notifications=默认 SMART 属性通知
 Default SMART controller type=默认 SMART 控制器类型
 Default SMART notification tolerance level=默认 SMART 通知容限级别
 Default SMART notification value=默认 SMART 通知值
-Default spin down delay=默认降速延迟
+Default spin down delay=默认休眠延迟
```
Comment (spencerjunraid) [inline settings.txt:120]: Keep original translation please.

File: settings.txt [hunk 9]
```diff
@@ -117,13 +117,13 @@ Default SMART attribute notifications=默认 SMART 属性通知
...
 Default VM storage path=默认虚拟机存储路径
 Default warning disk temperature threshold=默认警告磁盘温度阈值
-Default warning disk utilization threshold=默认警告磁盘利用率阈值
+Default warning disk utilization threshold=默认警告磁盘容量阈值
```
Comment (spencerjunraid) [inline settings.txt:123]: Keep original translation please.

File: settings.txt [hunk 10]
```diff
@@ -117,13 +117,13 @@ Default SMART attribute notifications=默认 SMART 属性通知
...
 Default Windows VirtIO driver ISO=默认 Windows VirtIO 驱动程序 ISO
 Define a TLS certificate=定义 TLS 证书
-Delete Image File=删除映像文件
+Delete Image File=删除镜像文件
```
Comment (spencerjunraid) [inline settings.txt:126]: Keep original translation please.

File: settings.txt [hunk 11]
```diff
@@ -504,7 +504,7 @@ Workgroup=工作组
 Write corrections to parity disk=写入校正至奇偶校验磁盘
 WSD options experimental=WSD 选项（实验性）
 Yearly=每年
-Your existing Docker image file needs to be recreated due to an issue from an earlier beta of Unraid 6=由于 Unraid 6 早期测试版中一个问题，需要重新创建现有 Docker 映像文件
+Your existing Docker image file needs to be recreated due to an issue from an earlier beta of Unraid 6=由于 Unraid 6 早期测试版中一个问题，需要重新创建现有 Docker 镜像文件
```
Comment (spencerjunraid) [inline settings.txt:507]: Keep original translation please.

File: helptext.txt [hunk 12]
```diff
@@ -1539,13 +1539,13 @@ Unraid 涵盖广受欢迎的 `vsftpd` FTP 服务器。`vsftp` 的设置现在十
 第三个数据包在第一个从属接口中发送，依此类推。
 该模式提供负载平衡和容错。
 
-**模式 1（主动备份） - 默认**<br>
-该模式将其中的一个接口置于备份状态，并且只在主动接口丢失链路时激活该接口。
-在某个时间实例内，该绑定中只有一个从属接口处于活动状态。只有在活动的从属接口出现故障时，才会激活另一个从属接口。
+**模式 1（主从模式） - 默认**<br>
+该模式将其中的一个接口置于备份状态，并且只在主接口丢失链路时激活该接口。
+在某个时间实例内，该绑定中只有一个设置的接口处于活动状态。只有在设置的主接口出现故障时，才会激活设置的备份接口。
```
Comment (spencerjunraid) [inline helptext.txt:1544]: 1542-1544: good.

File: helptext.txt [hunk 13]
```diff
@@ -1539,13 +1539,13 @@ Unraid 涵盖广受欢迎的 `vsftpd` FTP 服务器。`vsftp` 的设置现在十
...
 该模式提供容错。
 
 **模式 2（平衡 - 异或）**<br>
-该模式基于异或公式传输数据包。源 MAC 地址与目标 MAC 地址通过异或公式对从属接口数取模。
+该模式基于异或公式传输数据包。源 MAC 地址与目标 MAC 地址通过异或公式对从属接口数取模。(类似于ip hash, 只不过是用mac地址计算)
```
Comment (spencerjunraid) [inline helptext.txt:1548]: keep original please.

File: helptext.txt [hunk 14]
```diff
@@ -361,7 +361,7 @@ UnRAID 阵列由一个或两个奇偶校验磁盘和多个数据磁盘组成。
 :selftest_short_test_help:
 开始*快速* SMART 自检，可以在*功能*部分下查看预计持续时间。这通常是几分钟。
 
-当磁盘降速后，将中止所有正在运行的自检。
+当磁盘休眠后，将中止所有正在运行的自检。
```
Comment (spencerjunraid) [inline helptext.txt:364]: Keep original translation please.

File: helptext.txt [hunk 15]
```diff
@@ -225,17 +225,17 @@ UnRAID 阵列由一个或两个奇偶校验磁盘和多个数据磁盘组成。
...
 
-值为 0 将禁用警告阈值（包括通知）。
+值为 0 将禁用容量警告阈值（包括通知）。
 :end
 
 :info_critical_utilization_help:
 *危险磁盘利用率*设置此硬盘利用率的危险阈值。超过此阈值将导致警报通知。
 
-值为 0 将禁用危险阈值（包括通知）。
+值为 0 将禁用容量危险阈值（包括通知）。
```
Comment (spencerjunraid) [inline helptext.txt:238]: Keep original translation please.

File: helptext.txt [hunk 16]
```diff
@@ -225,17 +225,17 @@ UnRAID 阵列由一个或两个奇偶校验磁盘和多个数据磁盘组成。
 :end
 
 :info_warning_utilization_help:
-*警告磁盘利用率*设置此硬盘利用率的警告阈值。超过此阈值将导致警告通知。
+*警告磁盘利用率*设置此硬盘容量的警告阈值。超过此阈值将导致警告通知。
 
 当警告阈值设置等于或大于危险阈值时，将仅发出危险通知（不发出警告）。
 
-值为 0 将禁用警告阈值（包括通知）。
+值为 0 将禁用容量警告阈值（包括通知）。
```
Comment (spencerjunraid) [inline helptext.txt:232]: Keep original translation please.

File: helptext.txt [hunk 17]
```diff
@@ -225,17 +225,17 @@ UnRAID 阵列由一个或两个奇偶校验磁盘和多个数据磁盘组成。
 :end
 
 :info_warning_utilization_help:
-*警告磁盘利用率*设置此硬盘利用率的警告阈值。超过此阈值将导致警告通知。
+*警告磁盘利用率*设置此硬盘容量的警告阈值。超过此阈值将导致警告通知。
```
Comment (spencerjunraid) [inline helptext.txt:228]: Keep original translation please.

File: helptext.txt [hunk 18]
```diff
@@ -205,13 +205,13 @@ UnRAID 阵列由一个或两个奇偶校验磁盘和多个数据磁盘组成。
 :info_warning_temp_help:
 *警告磁盘温度*设置此硬盘温度的警告阈值，超过此阈值将导致警告通知。
 
-值为 0 将禁用警告阈值（包括通知）。
+值为 0 将禁用温度警告阈值（包括通知）。
```
Comment (spencerjunraid) [inline helptext.txt:208]: Keep original translation please.

File: helptext.txt [hunk 19]
```diff
@@ -193,7 +193,7 @@ UnRAID 阵列由一个或两个奇偶校验磁盘和多个数据磁盘组成。
 请参考 [cryptsetup FAQ]（https://gitlab.com/cryptsetup/cryptsetup/wikis/FrequentlyAskedQuestions#5-security-aspects）
 以了解构成*安全*密码的元素。
 
-**记住**这个密码。**如果丢失密码，已加密的内容将无法恢复！**
+**牢记**这个密码。**如果丢失密码，已加密的内容将无法恢复！**
```
Comment (spencerjunraid) [inline helptext.txt:196]: Keep original translation please.

File: helptext.txt [hunk 20]
```diff
@@ -69,7 +69,7 @@ UnRAID 阵列由一个或两个奇偶校验磁盘和多个数据磁盘组成。
 
 <i class='fa fa-circle orb green-orb'></i>正常运行，设备处于可用状态。
 
-<i class='fa fa-circle orb grey-orb'></i>设备处于待机模式（旋转降速）。
+<i class='fa fa-circle orb grey-orb'></i>设备处于待机模式（休眠）。
```
Comment (spencerjunraid) [inline helptext.txt:72]: Keep original translation please.

File: helptext.txt [hunk 21]
```diff
@@ -40,7 +40,7 @@
 **查看**列中有一个文件夹图标，表示设备已*挂载*。单击此图标
 浏览文件系统。
 
-如果在 [显示首选项] (Settings/DisplaySettings) 页面开启了 "显示阵列总计"，则
+如果在 [设置/显示首选项] (Settings/DisplaySettings) 页面开启了 "显示阵列总计"，则
```
Comment (spencerjunraid) [inline helptext.txt:43]: Keep original translation please.

File: helptext.txt [hunk 22]
```diff
@@ -16,9 +16,9 @@
 **标识**是用于唯一识别存储设备的*签名*。签名
 包括设备型号、序列号、Linux 设备 ID 和设备大小。
 
-**温度**直接从设备读取。您可以在 [显示首选项] (Settings/DisplaySettings)
-页面配置要使用的单位。我们不读取降速硬盘驱动器的温度，
-因为这通常会使它们加速旋转，所以改为显示 `*` 符号。我们还会显示 `*` SSD 和闪存设备的符号，
+**温度**直接从设备读取。您可以在 [设置/显示首选项] (Settings/DisplaySettings)
+页面配置要使用的单位。我们不读取休眠硬盘驱动器的温度，
+因为这通常会使它们重新激活，所以改为显示 `*` 符号。我们还会显示 `*` SSD 和闪存设备的符号，
```
Comment (spencerjunraid) [inline helptext.txt:21]: 19-21: Keep original translation please.

File: helptext.txt [hunk 23]
```diff
@@ -3,7 +3,7 @@
 
 <i class='fa fa-circle orb green-orb'></i>正常运行，设备处于活动状态。
 
-<i class='fa fa-circle orb grey-orb'></i>设备处于待机模式（旋转降速）。
+<i class='fa fa-circle orb grey-orb'></i>设备处于待机模式（休眠）。
```
Comment (spencerjunraid) [inline helptext.txt:6]: Keep original translation please.

File: main.txt [hunk 24]
```diff
@@ -12,11 +12,11 @@ Balance Status=余额状态
...
 Blacklisted=已列入黑名单
 Boot Device=引导设备
-Boot system in UEFI mode=UEFI 模式下的引导系统
-btrfs balance status=btrfs 余额状态
+Boot system in UEFI mode=以 UEFI 模式引导系统
+btrfs balance status=btrfs 均衡状态
 btrfs check status=btrfs 检查状态
 btrfs filesystem df=btrfs 文件系统 df
-BTRFS operation is running=BTRFS操作正在运行
+BTRFS operation is running=BTRFS文件系统正在执行操作
```
Comment (spencerjunraid) [inline main.txt:19]: Please change to BTRFS 操作正在运行

File: main.txt [hunk 25]
```diff
@@ -27,8 +27,8 @@ Capabilities=功能
 Check Filesystem Status=检查文件系统状态
 Clear Stats=清除统计信息
 Clearing in progress=正在清除中
-Click to spin down device=点击以便设备提速
-Click to spin up device=点击以便设备降速
+Click to spin down device=点击使设备休眠
```
Comment (spencerjunraid) [inline main.txt:30]: PLease change to 点击以令设备降速

File: main.txt [hunk 26]
```diff
@@ -27,8 +27,8 @@ Capabilities=功能
...
+Click to spin up device=点击以便设备激活
```
Comment (spencerjunraid) [inline main.txt:31]: please change to 点击以令设备提速

File: main.txt [hunk 27]
```diff
@@ -191,43 +189,42 @@ SMART Settings=SMART 设置
...
-Spin Up=提速
-Spinup groups=提速组
+Spin down delay=硬盘休眠延迟
+Spin Down=休眠
+Spin Up=激活
+Spinup groups=激活组
 Started, array protected=已启动，阵列受保护
 Started, array unprotected=已启动，阵列未受保护
-Started, formatting=已启动，格式化
+Started, formatting=已启动，格式化中
```
Comment (spencerjunraid) [inline main.txt:198]: Please change to 已启动，正在格式化

File: main.txt [hunk 28]
```diff
@@ -191,43 +189,42 @@ SMART Settings=SMART 设置
...
-The array will be immediately available, but unprotected until Parity-Sync completes=阵列将立即可用，但**未受保护**，这种状态将延续到奇偶校验同步完成为止
+The array will be immediately available, but unprotected until Parity-Sync completes=阵列将立即可用，但**未受保护**，直到奇偶校验同步完成为止
 The replacement disk must be as big or bigger than the original=替换磁盘的容量必须大于或等于原始磁盘
 This is typically done when a new disk is added to the array to get it ready for files to be written to it=当将新磁盘添加到阵列中以使其准备好将文件写入其中时，就可以完成此操作
 This unRAID Server OS release has been withdrawn and may no longer be used. Please %s update your server=此 unRAID Server OS 版本已撤消，可能不再使用。请 %s 更新您的服务器
 This will reboot the system=这将重启系统
 This will shutdown the system=这将关闭系统
 This will stop the array=这将停止阵列
-This will stop the running operation=这将停止运行操作
+This will stop the running operation=这将停止进行中的操作
```
Comment (spencerjunraid) [inline main.txt:213]: Please change to 这将停止正在运行的操作

File: main.txt [hunk 29]
```diff
@@ -248,29 +245,29 @@ Warning disk temperature threshold=警告磁盘温度阈值
 Warning disk utilization threshold=警告磁盘利用率阈值
 WARNING=警告
 Warranty period=保修期限
-will activate a clean system power down=将激活系统正常关机
-will activate a clean system reset=将激活系统正常重置
-will bring the array on-line and start Clearing new data disks=将使阵列联机并开始**清除**新数据磁盘
-will bring the array on-line and start Parity-Check=将使阵列联机并开始**奇偶校验**
-will bring the array on-line and start Parity-Sync andor Data-Rebuild=将使阵列联机并启动**奇偶校验同步**和/或**数据重建**
-will bring the array on-line=将使阵列联机
+will activate a clean system power down=将执行系统还原和关机
```
Comment (spencerjunraid) [inline main.txt:248]: Please update to 将执行正常系统关机

File: main.txt [hunk 30]
```diff
@@ -248,29 +245,29 @@ Warning disk temperature threshold=警告磁盘温度阈值
...
+will activate a clean system reset=将执行系统还原和重启
```
Comment (spencerjunraid) [inline main.txt:249]: please update to 将执行正常系统重启

File: main.txt [hunk 31]
```diff
@@ -248,29 +245,29 @@ Warning disk temperature threshold=警告磁盘温度阈值
 Warning disk utilization threshold=警告磁盘利用率阈值
 WARNING=警告
 Warranty period=保修期限
-will activate a clean system power down=将激活系统正常关机
-will activate a clean system reset=将激活系统正常重置
-will bring the array on-line and start Clearing new data disks=将使阵列联机并开始**清除**新数据磁盘
```
Comment (spencerjunraid) [inline main.txt:253]: Please change to 将使阵列上线并开始**清除**新数据磁盘

File: main.txt [hunk 32]
```diff
@@ -248,29 +245,29 @@ Warning disk temperature threshold=警告磁盘温度阈值
...
-will bring the array on-line and start Parity-Check=将使阵列联机并开始**奇偶校验**
```
Comment (spencerjunraid) [inline main.txt:254]: Please change to 将使阵列上线并开始**奇偶校验**

File: main.txt [hunk 33]
```diff
@@ -248,29 +245,29 @@ Warning disk temperature threshold=警告磁盘温度阈值
...
-will bring the array on-line and start Parity-Sync andor Data-Rebuild=将使阵列联机并启动**奇偶校验同步**和/或**数据重建**
```
Comment (spencerjunraid) [inline main.txt:255]: Please change to 将使阵列上线并开始**奇偶校验同步**和/或**数据重建**

File: main.txt [hunk 34]
```diff
@@ -248,29 +245,29 @@ Warning disk temperature threshold=警告磁盘温度阈值
...
-will bring the array on-line=将使阵列联机
```
Comment (spencerjunraid) [inline main.txt:256]: Please change to 将使阵列上线

File: main.txt [hunk 35]
```diff
@@ -248,29 +245,29 @@ Warning disk temperature threshold=警告磁盘温度阈值
...
+will activate a clean system reset=将执行系统还原和重启
+will bring the array on-line and start Clearing new data disks=将使阵列上线并**清除**新数据磁盘
+will bring the array on-line and start Parity-Check=将使阵列上线并启用**奇偶校验**
+will bring the array on-line and start Parity-Sync andor Data-Rebuild=将使阵列上线并启动**奇偶校验同步**和/或**数据重建**
+will bring the array on-line=将使阵列上线
 will copy the parity information to the new parity disk=将奇偶校验信息复制到新的奇偶校验磁盘
-will create a file system in all Unmountable disks=将在所有不可挂载的磁盘中创建文件系统
+will create a file system in all Unmountable disks=将在所有无法挂载的磁盘中创建文件系统
 will delete the encryption keyfile=将删除加密密钥文件
-will disable the missing disk and then bring the array on-line Install a replacement disk as soon as possible=将禁用缺失的磁盘，然后使阵列联机。尽快安装替换磁盘
```
Comment (spencerjunraid) [inline main.txt:260]: Please update to  将禁用缺失的磁盘，然后使阵列上线。请尽快安装替换磁盘

File: main.txt [hunk 36]
```diff
@@ -248,29 +245,29 @@ Warning disk temperature threshold=警告磁盘温度阈值
...
-will expand the file system of the data disk if possible then bring the array on-line and start Data-Rebuild=将扩展数据磁盘的文件系统；如果可能，则将阵列联机并开始数据重建
```
Comment (spencerjunraid) [inline main.txt:261]: Please change to 如果可能，将会扩展数据磁盘的文件系统后将阵列上线并开始数据重建

File: main.txt [hunk 37]
```diff
@@ -248,29 +245,29 @@ Warning disk temperature threshold=警告磁盘温度阈值
...
-will immediately spin up all disks=将立即对所有磁盘提速
+will immediately spin down all disks=将立即对所有磁盘硬盘休眠
+will immediately spin up all disks=将立即对所有磁盘硬盘活动
 will pause the Clearing=将暂停清除
 will pause the Parity-Check=将暂停奇偶校验
 will pause the Parity-SyncData-Rebuild=将暂停奇偶校验同步/数据重建
 will pause the Read-Check=将暂停读取-检查
-will record all disk information and bring the array on-line=将记录所有磁盘信息并使阵列联机
-will record all disk information, bring the array on-line, and start Parity-Sync=将记录所有磁盘信息，使阵列联机并启动奇偶校验-同步
-will record the new disks information and bring the expanded array on-line=将记录新磁盘信息并将扩展的阵列联机
```
Comment (spencerjunraid) [inline main.txt:272]: Please update to 将记录新磁盘信息并将已扩展的阵列上线

**Review by spencerjunraid — CHANGES_REQUESTED** [review]
Have reviewed with @fss003124. Have made comments on changes requested. lines 15-16,65-66,151,206,230,258,270-271,273,287 are all good.

Lines 141-142,194-197,219,226,228,264-265, please stick with original

File: helptext.txt (hunk 22 above)
Comment (jinnyu) [reply helptext.txt:21]: The displayed path is different, should I change it back to the original text?
Other will back to original text.

File: main.txt (hunk 28 above)
Comment (jinnyu) [reply main.txt:213]: Unnecessary suggestion

File: main.txt (hunk 29 above)
Comment (jinnyu) [reply main.txt:248]: The description here is inconsistent with the existing behavior of unraid.
The action taken by unraid now is to restore some settings (for example: reset SSH related configuration / reset FTP service, etc.),
So I translate "clean" here as 还原.
My translation has certain problems, but the "clean" here is better for accurate translation.

File: main.txt (hunk 30 above)
Comment (jinnyu) [reply main.txt:249]: The description here is inconsistent with the existing behavior of unraid.
The action taken by unraid now is to restore some settings (for example: reset SSH related configuration / reset FTP service, etc.),
So I translate "clean" here as 还原.
My translation has certain problems, but the "clean" here is better for accurate translation.

File: main.txt (hunk 28 above)
Comment (fss003124) [reply main.txt:213]: Hi jinnyu, 
Thanks for the effort!
I was trying to unifying the translation.. In this case, '-ing' wording to '正在xx'..
As the very first original work was handled by multiple people (myself included), it often times got inconsistence.. 
So yeah... Hope you understand. And thanks again for your contributions

Comment (jinnyu) [reply main.txt:213]: OK

**Comment by spencerjunraid** [comment]
Hello,  Changes to this PR and conflicts with the master branch exist: https://github.com/unraid/lang-zh_CN/pull/58/conflicts

Please update and resubmit PR.

**Pull request closed without merging**